src/index.js

```diff
@@ -29,7 +30,9 @@ const Root = () => {
   return (
     <MuiThemeProvider>
       <Provider store={store}>
-        <ApplicationPage history={history} />
+        <IntlProvider locale="en">
```

Comment:
It is required by redux-form-validators.  It is used for creating FormattedMessages which are used as defaultMessages by the redux-form-validators.